src/mpid/ch4/src/ch4_init.c

```diff
@@ -466,13 +466,31 @@ static int generic_init(void)
 
 int MPID_Init(int requested, int *provided)
 {
-    int mpi_errno = MPI_SUCCESS, rank, size, appnum;
-    MPIR_Comm *init_comm = NULL;
-    char strerrbuf[MPIR_STRERROR_BUF_SIZE];
-
+    int mpi_errno = MPI_SUCCESS;
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPID_INIT);
```

Comment:
Can we separate the code changes of ch3 and ch4 into two commits?

Reply:
Yes.

Reply:
Done.